src/app/modules/myinfo/myinfo.types.ts

```diff
@@ -23,9 +23,9 @@ export interface IMyInfoRedirectURLArgs {
 // Field ID or a special key for a Child
 export type MyInfoKey = string | MyInfoChildKey
 
-// Field type, field ID, child attribute type, child index, child name
+// Field type, field ID, child attribute type, child name
 export type MyInfoChildKey =
-  `${MyInfoAttribute.ChildrenBirthRecords}.${string}.${MyInfoChildAttributes}.${number}.${string}`
```

Comment:
minor clarification: why remove the number here; would this not prevent the (admittedly minor) change of key collision between child records? e.g. twins with the same name 😅

Reply:
thanks for this comment @tshuli ! it made me realise the case where there's >1 child per child field (when the respondent chooses to "add another child"). Previously I assumed 1 child per child field. I've added the index back here [d173907](https://github.com/opengovsg/FormSG/pull/6745/commits/d1739075b59d65998b0b0ea73afa73f0af57b578)